portal/src/components/Materials/SortableMaterials/SortableMaterials.component.html

```diff
@@ -0,0 +1,141 @@
+<section class="collection__materials">
+  <slot name="header-info"></slot>
+  <draggable
+    v-model="myList"
+    draggable=".materials__item"
+    class="materials__items deleting"
+    :class="{
+      loading: current_loading,
+      list: itemsInLine === 1,
+      tile: itemsInLine > 1
+    }"
+  >
+    <div
+      v-for="material in myList"
+      :class="[
+        `tile--items-in-line-${itemsInLine}`,
+        `materials__item--items-in-line-${itemsInLine}`,
+        'select-delete'
+      ]"
+      :key="material.external_id"
+      class="materials__item tile"
+    >
+      <div class="materials__bookmark" v-if="material.has_bookmark">Bookmark</div>
+      <button
+        class="select-icon"
+        v-if="contentEditable"
+        @click="deleteFromCollection(material)"
+      >
+        Delete
+      </button>
+      <div
+        :class="{'selected': material.selected}"
+        class="materials__item_wrapper tile__wrapper"
+        @click="handleMaterialClick(material)"
+      >
+        <div class="materials__item_content">
```

Comment:
Hebben we dit hele stuk niet op meerdere plekken? Is het niet een idee om dan een Material component oid te maken?

Reply:
dat is dus waar ik gisteren mee bezig was wat niet goed lukte ;-)

Reply:
maar nu wel gelukt